routes/users.go

```diff
@@ -28,4 +30,42 @@ func UserRoutes(rg *gin.RouterGroup, db *bun.DB) {
 			"data":    users,
 		})
 	})
+
+	users.GET("/:id",func(ctx *gin.Context) {
+		id := ctx.Param("id")
+
+		var user models.User
+		err := db.NewSelect().Model(&user).Where("id = ?", id).Scan(ctx)
+
+		if err != nil {
+			ctx.JSON(http.StatusInternalServerError, gin.H{
+				"message": "error",
+			})
+			return
+		}
+
+		ctx.JSON(http.StatusOK, gin.H{
+			"message": "user fetched successfully",
+			"data":    user,
+		})
+	})
+
+	users.GET("/self", func(ctx *gin.Context) {
+		userEmail, _ := ctx.Get("user")
+
+		var user models.User
+		err := db.NewSelect().Model(&user).Where("email = ?", userEmail).Scan(ctx)
+
+		if err != nil {
+			ctx.JSON(http.StatusInternalServerError, gin.H{
+				"message": "error",
+			})
+			return
+		}
+
+		ctx.JSON(http.StatusOK, gin.H{
+			"message": "user fetched successfully",
+			"data":    user,
+		})
+	})
```

Comment:
Make a controller file and move this to the controller file

Reply:
Done